<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplatesConfigurationBase</class>
 <widget class="QWidget" name="TemplatesConfigurationBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>541</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>3</horstretch>
    <verstretch>3</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string comment="@title:window">Template Configuration</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin">
    <number>11</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox1">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>3</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_new">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>350</height>
       </rect>
      </property>
      <attribute name="label">
       <string comment="@title Message template">New Message</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="TemplateParser::TemplatesTextEdit" name="textEdit_new">
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>9</pointsize>
          </font>
         </property>
         <property name="toolTip">
          <string comment="@info:tooltip">Create the template for new messages</string>
         </property>
         <property name="whatsThis">
          <string comment="@info:whatsthis">In this area you create the template for new email messages.  For more information about how to create the template, press the &quot;How does this work?&quot; link on this dialog.</string>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_reply">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>350</height>
       </rect>
      </property>
      <attribute name="label">
       <string comment="@title Message template">Reply to Sender</string>
      </attribute>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="TemplateParser::TemplatesTextEdit" name="textEdit_reply">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="toolTip">
          <string comment="@info:tooltip">Create the template for message replies</string>
         </property>
         <property name="whatsThis">
          <string comment="@info:whatsthis">In this area you create the template for messages replies. For more information about how to create the template, press the &quot;How does this work?&quot; link on this dialog.</string>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_reply_all">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>350</height>
       </rect>
      </property>
      <attribute name="label">
       <string comment="@title Message template">Reply to All / Reply to List</string>
      </attribute>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="TemplateParser::TemplatesTextEdit" name="textEdit_reply_all">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="toolTip">
          <string comment="@info:tooltip">Create the template for replying to all recipients or to a mailing list</string>
         </property>
         <property name="whatsThis">
          <string comment="@info:whatsthis">In this area you create the template for reply-to-all messages or replies to a mailing list.  For more information about how to create the template, press the &quot;How does this work?&quot; link on this dialog.</string>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_forward">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>418</width>
        <height>350</height>
       </rect>
      </property>
      <attribute name="label">
       <string comment="@title Message template">Forward Message</string>
      </attribute>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="TemplateParser::TemplatesTextEdit" name="textEdit_forward">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Monospace</family>
           <pointsize>9</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="toolTip">
          <string comment="@info:tooltip">Create the template for message forwards</string>
         </property>
         <property name="whatsThis">
          <string comment="@info:whatsthis">In this area you create the template for message forwards.  For more information about how to create the template, press the &quot;How does this work?&quot; link on this dialog.</string>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="mHelp"/>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="TemplateParser::TemplatesInsertCommand" name="mInsertCommand" native="true"/>
     </item>
     <item>
      <widget class="QLabel" name="textLabel1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string comment="@label:textbox Prefix for quoted message lines">&amp;Quote indicator:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="buddy">
        <cstring>lineEdit_quote</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KLineEdit" name="lineEdit_quote">
       <property name="toolTip">
        <string comment="@info:tooltip">Set the Prefix for quoted message lines</string>
       </property>
       <property name="whatsThis">
        <string comment="@info:whatsthis">
             &lt;qt&gt;The following placeholders are supported in the quote indicator:
             &lt;ul&gt;
             &lt;li&gt;%f: sender's initials&lt;/li&gt;
             &lt;li&gt;%%: percent sign&lt;/li&gt;
             &lt;li&gt;%_: space&lt;/li&gt;
             &lt;/ul&gt;&lt;/qt&gt;
           </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>TemplateParser::TemplatesTextEdit</class>
   <extends>QTextEdit</extends>
   <header>templatestextedit.h</header>
  </customwidget>
  <customwidget>
   <class>TemplateParser::TemplatesInsertCommand</class>
   <extends>QWidget</extends>
   <header>templatesinsertcommand.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
